pkg/initialdata/default-initial-data.json

```diff
@@ -1,5 +1,5 @@
 {
-    "_migration_index": -1,
+    "_migration_index": 3,
```

Comment:
Is there a reason why it is 3? Maybe add a comment that explains it

Reply:
See main repository.